src/fasthash.c

```diff
@@ -27,10 +27,13 @@
 
 // Compression function for Merkle-Damgard construction.
 // This function is generated using the framework provided.
-#define mix(h) ({					\
-			(h) ^= (h) >> 23;		\
-			(h) *= 0x2127599bf4325c37ULL;	\
-			(h) ^= (h) >> 47; })
+static inline uint64_t mix(uint64_t h)
+{ /* use inline function instead of non-standard macro */
```

Comment:
Let's kill this comment.  After the change has been made, it seems unlikely that anyone will go out of their way to put the GCC extension back in place even if the comment is not present.